app/models/alchemy/page.rb

```diff
@@ -118,6 +118,9 @@ class Page < ActiveRecord::Base
       if: -> { public_on.present? && published_at.nil? },
       unless: :systempage?
 
+    before_save :set_fixed_atrributes,
+      if: -> { fixed_attributes.any? }
```

Comment:
The question here should really be: Are there any, and did anyone try to change them? I'm uneasy with setting the attrs on every save, as they really should only be touched once when creating the page, and then never again. 

Also: The name of the callback has a spelling mistake ;)